include/openspace/engine/openspaceengine.h

```diff
@@ -101,20 +102,78 @@ class OpenSpaceEngine {
     void writeStaticDocumentation();
     void createUserDirectoriesIfNecessary();
 
+    /**
+     * Sets the camera position using the time contents of a profile. The function will
+     * set an absolute position or a go-to-geolocation command using the globebrowsing
+     * module.
+     * \param p The Profile to be read.
+     */
+    void setFromProfile_camera(const Profile& p);
```

Comment:
We're not using `_` anywhere in functions really, so maybe:  `setCameraFromProfile` would be better?  Same for the other functions below

Reply:
Most of the functions also don't do anything with the OpenSpaceEngine, so they don't have to be functions of that class at all and can just be defined locally in the cpp file, too

Reply:
Changed these names and made them independent functions.